### ISSUE TITLE
Acquia key and secret via environment variable

### ISSUE THREAD

**Issue by shelane** [typhonius/acquia_cli#29]
I would like to add this as part of a docker-based environment where each member of our team has their key and secret stored as an environment variable. We would not want a file containing that information to be placed into the git repository.

Will it read it if the file is set as:
```yml
acquia:
  key: ${SECRET_ACAPI_KEY}
  secret: ${SECRET_ACAPI_SECRET}
```

Or is there another way to accomplish the same thing?

**Comment by typhonius** [comment]
This is a good idea, we currently use an approach which inherits config from global, then local. I think we could extend that to check for an environment variable to feed to the SDK.

**Comment by typhonius** [comment]
Can you check #32 to see if the changes are fit for purpose. I've set the environment variables to be the following so there is no chance of clashing with others:
* ACQUIACLI_KEY
* ACQUIACLI_SECRET

**Comment by shelane** [comment]
Looks good. So if the environment variable were set, it would override whatever is in the yml file?

**Comment by typhonius** [comment]
Only the key and secret. This would mean you could still commit any other configuration to an acquiacli.yml in the project directory eg timezone, task wait timeout etc

**Comment by typhonius** [comment]
Merged in 13444786c49d69e8ded6b9530d7d44727a125e89
